src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/BasicModelDefTest.java

```diff
@@ -150,5 +150,13 @@ public void globalLibrarySuccess() throws Exception {
 
         j.assertLogContains("[nothing here]", b);
         j.assertLogContains("map call(1,2)", b);
+        /* TODO called the wrong way: closure1([a:1])
```

Comment:
So this is getting called [here](https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/6f7cf8ed718fde0e8da3b31ed1235b15fbe6bd37/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/ModelInterpreter.groovy#L76-L79) - we're delegating the `Closure` to the `CpsScript` instance. I honestly don't know why that would be different in execution than normal, but it apparently is? Any ideas?